shell/platform/fuchsia/flutter/component_v1.cc

```diff
@@ -62,15 +65,27 @@ std::string DebugLabelForUrl(const std::string& url) {
 void ComponentV1::ParseProgramMetadata(
     const fidl::VectorPtr<fuchsia::sys::ProgramMetadata>& program_metadata,
     std::string* data_path,
-    std::string* assets_path) {
+    std::string* assets_path,
+    int64_t* old_gen_heap_size) {
```

Comment:
This should really be std::optional<int64_t>, but the existing code is already using pointer so....

Reply:
Done using a struct with an optional. PTAL.